Chinese/Mr.X/你们真会玩儿

```diff
@@ -0,0 +1 @@
+从前有个山，山里有个庙，庙里有个小和尚和一个老和尚，小和尚对老和尚说，爸！吃饭啦！
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.